server/selinux/server/pulp-celery.te

```diff
@@ -52,6 +57,12 @@ allow celery_t tmpfs_t:filesystem getattr;
 allow celery_t tmpfs_t:dir { write add_name remove_name };
 allow celery_t tmpfs_t:file { create open read write link getattr unlink };
 
+
+#============= qpidd_t ===============
+#
+
+allow qpidd_t qpidd_var_lib_t:file map;
```

Comment:
Let's leave out the qpidd ones. I suspect this failure didn't cause Pulp failures, but it prevented Qpid code from doing something it tried to do. If Pulp still has an issue after these other changes are merged, then we can raise the issue with them. Sound ok?